django-s3-file-field-client/src/client.ts

```diff
@@ -120,11 +134,12 @@ export default class S3FFClient {
    * Uploads a file using multipart upload.
    *
    * @param file the file to upload
+   * @param fieldId the django model field's id
    * @param options the upload options
    */
-  public async uploadFile(file: File, fieldId: string): Promise<UploadResult> {
+  public async uploadFile(file: File, fieldId: string, options: UploadOptions = {}): Promise<UploadResult> {
```

Comment:
I think for now, `options` is premature. I'd rather expose this directly as an optional parameter. I'm open to counterargument though.